app/models/user.rb

```diff
@@ -13,6 +14,36 @@ class User < ApplicationRecord
   validates :password, length: {minimum: Settings.length.password}
   validate :valid_birthday, if: ->{birthday.present?}
 
+  class << self
+    # Returns the hash digest of the given string.
+    def digest string
+      cost = if ActiveModel::SecurePassword.min_cost
+               BCrypt::Engine::MIN_COST
+             else
+               BCrypt::Engine.cost
+             end
+      BCrypt::Password.create string, cost: cost
+    end
+
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
+  end
+
+  def authenticated? remember_token
+    BCrypt::Password.new(remember_digest).is_password? remember_token
+  end
+
+  # Forgets a user.
+  def forget
+    update_attribute :remember_digest, nil
```

Comment:
```suggestion
    update_column :remember_digest, nil
```
ri mwois là hợp lsy nhất nhé cu

Reply:
.